Makefile

```diff
@@ -67,6 +67,21 @@ publish_to_maven_local:
 publish_to_artifactory:
 	./gradlew -p subprojects publishToArtifactory -PprojectVersion=$(version) $(log_level)
 
+# precondition:
+# - installed CLI: https://cli.github.com/
+# - push $(version) branch
+#
+# post actions:
+# - go to link in output
+# - edit notes
+# - publish release
+draft_release:
```

Comment:
Please, mention this in https://avito-tech.github.io/avito-android/docs/release/

Reply:
done